common/web/types/src/package/kmp-json-file.ts

```diff
@@ -74,3 +81,27 @@ export interface KmpJsonFileStartMenuItem {
   icon?: string;
   location?: string;
 }
+
+export interface KmpJsonFileExample {
+  /**
+   * BCP 47 identifier for the example
+   */
+  id: string;
+  /**
+   * A space-separated list of keys, modifiers indicated with "+", spacebar is "space", plus key is "shift+=" or "plus"
+   */
+  keys: string;
+  /**
+   * The text that would be generated by typing those keys
+   */
+  text?: string;
+  /**
+   * A short description of what the text means or represents
+   */
+  note?: string;
+}
+
+export interface KmpJsonRelatedPackage {
+  id: string;
+  relationship: "deprecates" | "related";
```

Comment:
It's good to get away from maintaining .keyboard_info.

Is it still a manual process of adding DEPRECATED.md files for the applicable keyboards?

Reply:
Yes, because that's file is there in repo to inform keyboard devs that the keyboard in question is deprecated.